DjmaxRandomSelectorV/ViewModels/SettingViewModel.cs

```diff
@@ -16,6 +19,7 @@ public class SettingViewModel : Screen
         private List<string> _ownedDlcs;
 
         private DockPanel _dockPanel;
+        private int updated;
```

Comment:
좀 더 직관적인 변수명을 사용하면 좋을 것 같습니다.
```suggestion
private int dlcCount;
```